Packs/Core/Playbooks/playbook-Unprivileged_process_opened_a_registry_hive_README.md

```diff
@@ -1,12 +1,10 @@
-This playbook is designed to handle the 'Unprivileged process opened a registry hive' alert. 
-
-The playbook is designed to investigate and respond to an unprivileged process opening a registry hive. It examines the unprivileged process that triggered the alert, the command line, and searches for any additional suspicious Cortex XSIAM alerts using Mitre techniques in order to determine whether a remediation measure is required.
+This playbook is designed to handle the 'Unprivileged process opened a registry hive' alert.
 
 Playbook Stages:
 
 Investigation:
 
-- The playbook is designed to investigate and respond to unprivileged processes opening registry hives. It examines the unprivileged process that triggered the alert, the command line, and searches for additional suspicious Cortex XSIAM alerts within the same incident in order to determine whether a remediation measure is required.
+- The playbook examines the unprivileged process that triggered the alert, the command line, and searches for additional suspicious Cortex XSIAM alerts within the same incident in order to determine whether a remediation measure is required.
```

Comment:
Please separate the investigation actions into different points so it will be cleaer